README.md

```diff
@@ -23,6 +23,17 @@ This mod works with a single dll file but we also provide a way to downpatch the
 installing the latest Microsoft Visual C++ Redistributable version could fix it : https://aka.ms/vs/17/release/vc_redist.x64.exe.
 - The Windows console tends to freeze preventing you from sending or receiving any items. You must Alt+Tab, click on the console and press enter to refresh it.
 
+## Building Locally
+
+1. Clone the repository (`git clone https://github.com/Marechal-L/Dark-Souls-III-Archipelago-clieng.git`).
+2. Open `archipelago-client\archipelago-client.sln` in Visual Studio.
+3. [Install OpenSSL for Windows](https://slproweb.com/products/Win32OpenSSL.html).
```

Comment:
it took me a little while to figure out that you need SSL version 1.1 for this to work (at least that's what I needed on my PC). I am not sure if that's just something about my setup or if it's true for everyone though. If you happen to know I think it would be useful to include that in this instruction